tsconfig.json

```diff
@@ -1,18 +1,19 @@
 {
   "compilerOptions": {
-    "baseUrl": ".",
-    "declaration": false,
-    "esModuleInterop": true,
-    "jsx": "react-jsx",
-    "lib": ["dom", "esnext"],
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
+    "lib": ["DOM", "ES2022"],
+    "types": [],
+    "target": "ES2022",
     "outDir": "lib",
-    "removeComments": false,
-    "sourceMap": false,
+    "jsx": "react-jsx",
+    "module": "NodeNext",
```

Comment:
This is the change along with the `"type": "module"` change in `package.json` which forces you to import with file extensions, right?

Reply:
Yes.